### ISSUE TITLE
List of spec tests not covered by Lodestar

### ISSUE THREAD

**Issue by q9f** [ChainSafe/lodestar#3199]
I compared the tag release `v0.30.0` (alpha) which ran pipeline [3619666497](https://github.com/ChainSafe/lodestar/runs/3619666497) with the upstream spec test at tag `v1.1.0-beta.3`. I have put my findings in the linked Gists (if any).

### Phase 0

* `general/phase0/bls/` 60/60 (100%) ✅ 
* `general/phase0/ssz_generic/` 1795/1905 (94.2%, 110 [not covered](https://gist.github.com/q9f/3082ce2f694fe3050d42bc2f65c519a0))
* `mainnet/phase0/` 712/712 (100%) ✅ 
* `minimal/phase0/` 3912/3919 (99.8%, _7 [pending](https://gist.github.com/q9f/509fd97eed9ba29d174dbff2cf9a0e55)_)

I don't know what pending means in CI, they are implemented but not run? Or timeout?

### Altair

* `general/altair/bls/` 0/20 (0.0%, 20 [not covered](https://gist.github.com/q9f/7c504bad544ea47d524b10fb5a7d8dde))
* `mainnet/altair/` 459/504 (90.7%, 45 [not covered](https://gist.github.com/q9f/5120b4f50062f852e95bc0e6fa205e4a))
* `minimal/altair/` 3675/4784 (76.8%, _diff too large/unknown_)

It's not an easy process to create these diffs, so I did not look into the last large diff due to time constraints. Phase 0 and Mainnet should be prioritized though. Also the merge diffs are huge (obviously), so I didn't bother to list them.

**Comment by q9f** [comment]
It would be beneficial to quickly get a list of available tests in the consensus repository. So I used a ruby script to parse the contents and generate the lists.

Also, I couldn't use the TypeScript utilities that are supposed to check test coverage. So, instead, I parsed the CI logs to get a clear image of what's being run. Also, here, it would be nice if I could figure out a way in the future, to access the list of covered tests more quickly in the future.
